strawberryfields/engine.py

```diff
@@ -532,14 +526,16 @@ def run(self, program: Program, shots: Optional[int] = None) -> Optional[Result]
 
         Args:
             program (strawberryfields.Program): the quantum circuit
+
+        Keyword Args:
             shots (Optional[int]): The number of shots for which to run the job. If this
```

Comment:
How does this show up on rendered docs? Does the name `run_options` appear in signature, but here `shots` is listed as a kwarg (essentially just a renamed `kwargs`)

Reply:
Exactly!

Reply:
![image](https://user-images.githubusercontent.com/24476053/77779757-62190b80-7029-11ea-8d18-7a93c27d882c.png)

Reply:
Oops, looks like there is a formatting issue in the `Returns:` line!

Reply:
Unfortunately, attempting to change `Keyword Args` to anything not recognised by `sphinx.napoleon` leads to formatting/parsing issues:

![image](https://user-images.githubusercontent.com/2959003/77821925-e9f03a00-713d-11ea-86c2-970ef11d008f.png)

The other option is to just change `**runtime_options` to `**kwargs`?

Reply:
Have updated it to `**kwargs` in all places it appeared